Assets/HoloToolkit/Input/Scripts/InputSources/IInputSource.cs

```diff
@@ -119,6 +119,11 @@ public interface IInputSource
         event EventHandler<NavigationEventArgs> NavigationCanceled;
 
         /// <summary>
+        /// Event triggered when a speech phrase is recognized.
+        /// </summary>
+        event EventHandler<PhraseRecognizedEventArgs> PhraseRecognized;
```

Comment:
Yeah, I'm OK with leaving like that for now unless someone comes up with a better name. Makes sense to align with existing name for things.